fxmanifest.lua

```diff
@@ -3,8 +3,9 @@ game "gta5"
 description 'js runtime monkaW'
 authors { "itschip",  "erik-sn", "TasoOneAsia", "kidz", "RockySouthpaw"}
 
-client_script {
-    'resources/dist/client/*.client.js'
+client_scripts {
+    'resources/dist/client/*.client.js',
+    'futile.lua'
```

Comment:
You could just glob it as `*.lua` I guess and keep the meme name 🤷🏻‍♀️ but this is likely the only lua file to exist in the entire repo so.